docs/design/features/cet-feature.md

```diff
@@ -0,0 +1,28 @@
+
+## Opting into CET shadow stack compatibility
+
+CET compatibility is enabled in .NET 7. In order to receive the security benefits of CET shadow stacks in .NET applications, ensure the following to opt into the feature for an executable:
+- Verify that your processor supports Intel CET. There are Intel and AMD processors currently available with the feature.
+- Get a recent build of Windows that has Hardware-enforced Stack Protection. The latest versions from [Windows Insider preview builds (Dev channel)](https://insider.windows.com/en-us/understand-flighting) have the feature.
+- Install [.NET 7](https://dotnet.microsoft.com/download/dotnet) RC1 or a more recent release
+- Build or publish your application targeting framework `net7.0` and runtime `win-x64`
+- Open the Windows Security app
+  - Inside the "App & browser control" pane, select "Explicit protection settings"
+  - Under the "Program settings" tab, select "Add program to customize" and "Add by program name"
+  - Enter the file name of your application�s executable file and select "Add"
+  - Select the following options and select "Apply":
+
+    ![Hardware-enforced Stack Protection settings](images/intel-cet-dotnet6-fig1.png)
+  - See the [Developer Guidance for Hardware-enforced Stack Protection](https://techcommunity.microsoft.com/t5/windows-kernel-internals/developer-guidance-for-hardware-enforced-stack-protection/ba-p/2163340) for more details
+- Task Manager�s Details tab has a new column "Hardware-enforced Stack Protection" that shows the compatibility level of a running process. If CET shadow stacks are enabled for the process, the column for the application should say "Compatible modules only".
+
+## Considerations
+
+CET shadow stack compatibility in .NET is new within 7 and should work for most features an opt-in for x64 applications. 
+
+When opted in as above, CET shadow stacks are enabled in compatibility mode. .NET 7 is not yet fully compatible with strict mode.
```

Comment:
Is the compatibility mode that this talks about same as "Compatible modules only" in the previous paragraph?

If it is, I believe that the .NET runtime should work fine in strict mode. You only need the compatibility mode if your app is loading 3rd party native libraries that are not compatible with CET.